packages/react-aria-components/src/Toolbar.tsx

```diff
@@ -0,0 +1,176 @@
+/*
+ * Copyright 2023 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {ContextValue, forwardRefType, RenderProps, SlotProps, useContextProps, useRenderProps} from './utils';
+import {filterDOMProps, mergeProps} from '@react-aria/utils';
+import {FocusScope, useFocusManager} from '@react-aria/focus';
+import {MultipleSelection, Orientation} from '@react-types/shared';
+import React, {createContext, ForwardedRef, forwardRef, KeyboardEventHandler, useRef} from 'react';
+import {useLocale} from '@react-aria/i18n';
+
+export interface ToolbarRenderProps {
+  /**
+   * Whether the dropzone is currently hovered with a mouse.
+   * @selector [data-orientation]
+   */
+  orientation: Orientation
```

Comment:
Description for this render prop should be orientation specific, think just some left over copy pasta

Reply:
haha, yeah, some copy-pasta